README.md

```diff
@@ -14,21 +14,6 @@ Add this line to your application's Gemfile:
 gem 'mongoid_paranoia'
 ```
 
-## Changes in 4.0
```

Comment:
Hmm, do you have any link for that?

Anyway is 

```ruby
validates :title, :uniqueness => true
```

still valid?

If so, we need to keep this information and remove `validates_uniqueness_of` example only.